### ISSUE TITLE
Deeper analysis of data

### ISSUE THREAD

**Issue by trz-maier** [trz-maier/hwu-dmml-group1#4]
Searching for important attributes etc.

**Comment by trz-maier** [comment]
featuren importance shown as an image (lighter pixels are more important)
used ExtraTreesClassifier from sklearn.ensemble (not entirely sure this is right)
this is a result pre image processing

![image](https://user-images.githubusercontent.com/46493724/66716408-688e3080-edc5-11e9-8cdd-dfe934396135.png)

**Comment by trz-maier** [comment]
I have switched to using SelectKBest from sklearn.feature_selection with the following result

![image](https://user-images.githubusercontent.com/46493724/66716943-72b32d80-edcb-11e9-8032-3c9db3795225.png)

**Comment by trz-maier** [comment]
Looking for the optimal number of features for the model I found it to be somewhere around 500 mark (out of the total of 2304 features). This improves the accuracy slightly from 0.51 to 0.56 weighted average. Using all 2304 attributes allows to correctly predict 1110 of 3798 signs while filtering them down to 500 increases this number to 1494 out of 3798

![image](https://user-images.githubusercontent.com/46493724/66718430-d3e2fd00-eddb-11e9-9ab7-f07aa1ff0d1d.png)
